kakao/src/main/kotlin/com/agoda/kakao/list/KListView.kt

```diff
@@ -74,7 +75,7 @@ class KListView : ScrollViewActions, BaseAssertions, ListViewAdapterAssertions {
     @Suppress("UNCHECKED_CAST")
     constructor(parent: DataInteraction, builder: ViewBuilder.() -> Unit,
                 itemTypeBuilder: KAdapterItemTypeBuilder.() -> Unit) {
-        val makeTargetMatcher = DataInteraction::class.java.getDeclaredMethod("makeTargetMatcher")
+        val makeTargetMatcher = DataInteractionDelegate::class.java.getDeclaredMethod("makeTargetMatcher")
```

Comment:
This is reflection, delegate class does not have `makeTargetMatcher` function in it.